### PR TITLE
ACS-3545 a 11 y aca metadata sidebar

### PR DESCRIPTION
**Pull request by AleksanderSklorz**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Accessibility issues for metadata sidebar related with heading, missing aria-labels, missing labels for inputs and duplicated ids. 

**What is the new behaviour?**
Fixed all accessibility issues for metadata sidebar related with heading, missing aria-labels, missing labels for inputs and duplicated ids. 

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html [hunk 1]
```diff
@@ -5,8 +5,8 @@
                              [expanded]="canExpandProperties()"
                              [attr.data-automation-id]="'adf-metadata-group-properties'">
             <mat-expansion-panel-header>
-                <mat-panel-title role="heading">
```
Comment (MichalKinas) [inline lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html:8]: Why did you changed heading role to h1? Wouldn't be enough just to add aria-level in this case?

File: lib/core/src/lib/comments/comments.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="adf-comments-container">
-    <div id="comment-header" class="adf-comments-header" role="heading">
```
Comment (MichalKinas) [inline lib/core/src/lib/comments/comments.component.html:2]: Same here

**Review by MichalKinas — CHANGES_REQUESTED** [review]

File: lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html (hunk 1 above)
Comment (AleksanderSklorz) [reply lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html:8]: Yes, you are right that this is also a solution for that. There is reason why I decided to use h tag instead and if you think I should to use aria-level then let me know. That reason is described in following pages: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-level
"Opt for using the <h1> thru <h6> elements instead."

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role
"This defines the text in the <div> to be the main heading of the page, indicated by being level 1 via the aria-level attribute. Opt for using the <h1> thru [<h6>] element instead."

And also interesting explaining here, please read first answer: 
https://webmasters.stackexchange.com/questions/137159/for-seo-is-role-heading-aria-level-1-the-same-as-h1

Please let me know what do you think :)

File: lib/core/src/lib/comments/comments.component.html (hunk 2 above)
Comment (AleksanderSklorz) [reply lib/core/src/lib/comments/comments.component.html:2]: Please check my explanation in previous comment

File: lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html (hunk 1 above)
Comment (MichalKinas) [reply lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html:8]: Ok I reviewed the stuff you linked and I think this is fine :D Approved

**Review by MichalKinas — APPROVED** [review]

**Review by nikita-web-ua — APPROVED** [review]

**Review by DenysVuika — APPROVED** [review]

**Review by DenysVuika — APPROVED** [review]

**Comment by DenysVuika** [comment]
@AleksanderSklorz there are some e2e tests failing for this PR

**Comment by AleksanderSklorz** [comment]
@DenysVuika just pushed changes and hope that they will fix e2e. Very small corrections, mostly renaming selectors.

**Pull request merged**